src/pages/dashboards/MemberDashboardPage.jsx

```diff
@@ -13,6 +13,7 @@ import { useAuth } from '@/providers/AuthProvider';
 import { FeatureCard } from '@/compoents/FeatuerCards';
 import CardMedia from '@mui/material/CardMedia';
 import Box from '@mui/material/Box';
+import BasicCardComonent from '../../compoents/BasicCardComonent/BasicCardComonent';
```

Comment:
Maybe you should use an absolute import here? They're much easier to read and behave better when renaming files imo. 

```js
import BasicCardComponent from '@/components/...';
```